code/datums/quirks/_quirk.dm

```diff
@@ -10,7 +10,7 @@
 	/// Flags related to this quirk.
 	var/quirk_flags = QUIRK_HUMAN_ONLY
```

Comment:
Remove the flag if you're asserting quirks are only applied to humans (I don't even know why the flag exists tbh)